#! /bin/sh /usr/share/dpatch/dpatch-run
## Makefile.dpatch by  <hotzenplotz5@gmx.de>
##
##
## DP: patch needed for yaepghd to work with plugin-xine

@DPATCH@
diff -urNad xine-0.9.3~/Makefile xine-0.9.3/Makefile
--- xine-0.9.3~/Makefile	2009-10-02 12:28:51.000000000 +0200
+++ xine-0.9.3/Makefile	2009-10-02 12:49:13.976557490 +0200
@@ -45,7 +45,7 @@
 VDR_XINE_VERIFY_BITMAP_DIRTY ?= 0
 
 # enable to fully support yaepg plugin
-#VDR_XINE_SET_VIDEO_WINDOW = 1
+VDR_XINE_SET_VIDEO_WINDOW = 1
 
 # where are these utilities for image grabbing? (default: anywhere on your PATH)
 #VDR_XINE_Y4MSCALER = /usr/bin/y4mscaler
